From 25e1176b404eeb5f3402187ad4f854ccea95b801 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Mon, 24 Aug 2015 14:23:25 -0500
Subject: [PATCH 05/10] Revert "drm/omap: remove extra
 drm_gem_free_mmap_offset() call"

This reverts commit d1d4f63d4a5d19b1264f60f3957dfdf808709793.

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 drivers/gpu/drm/omapdrm/omap_gem.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/omapdrm/omap_gem.c b/drivers/gpu/drm/omapdrm/omap_gem.c
index f9ddbf5..7ed08fdc 100644
--- a/drivers/gpu/drm/omapdrm/omap_gem.c
+++ b/drivers/gpu/drm/omapdrm/omap_gem.c
@@ -1282,6 +1282,8 @@ void omap_gem_free_object(struct drm_gem_object *obj)
 	list_del(&omap_obj->mm_list);
 	spin_unlock(&priv->list_lock);
 
+	drm_gem_free_mmap_offset(obj);
+
 	/* this means the object is still pinned.. which really should
 	 * not happen.  I think..
 	 */
-- 
2.5.0

